Solutions/EatonForeseer/Analytic Rules/EatonUnautorizedLogins.yaml

```diff
@@ -0,0 +1,59 @@
+id: 5a7fccb8-3ed0-44f2-8477-540af3ef4d92
+name: EatonForeseer - Unauthorized Logins
+description: |
+  'Detects Unauthorized Logins into Eaton Foreseer'
+severity: High
+requiredDataConnectors:
+  - connectorId: WindowsSecurityEvents
+    dataTypes:
+      - SecurityEvent
+queryFrequency: 15m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1078
+query: |
+  SecurityEvent
+  // Look for any events in the last 24 hours
+  | where TimeGenerated > ago(1d)
```

Comment:
Since we already have a queryFrequency/queryPeriod, not sure if we need the TimeGenerated here. Can you please have a look.

Reply:
I was thinking about visibility, but it makes sense to remove it

Reply:
Thanks @kushaljh. Removing it is a better idea.